site/docs/v1/tech/events-webhooks/index.adoc

```diff
@@ -22,23 +22,24 @@ See the corresponding link:/docs/v1/tech/apis/webhooks[Webhook APIs] if you'd pr
 
 Here are the topics in this section:
 
-* link:/docs/v1/tech/events-webhooks/events/[Events] - Covers all of the event types that FusionAuth sends to Webhooks
 * link:/docs/v1/tech/events-webhooks/writing-a-webhook[Writing a Webhook] - Covers how to write a Webhook that will process events from FusionAuth.
 * link:/docs/v1/tech/events-webhooks/securing[Securing a Webhook] - Covers how to ensure your webhooks are secured properly.
+* link:/docs/v1/tech/events-webhooks/events/[Events] - Covers all of the event types that FusionAuth sends to Webhooks
 
 Continue reading below to see how the events and webhooks are configured using the FusionAuth user interface.
 
 * <<Tenant Settings>>
 * <<Add Webhook>>
 * <<Test a Webhook>>
+* <<FAQs>>
 
 == Tenant Settings
 
-To prepare to consume FusionAuth events you'll first need to enable the event and select a transaction level that is compatible with your requirements. 
+To prepare to consume FusionAuth events you'll first need to enable the event and select a transaction level that is compatible with your requirements in the tenant. 
 
 [NOTE]
 ====
-You do not need to configure the tenant settings for any instance level webhooks, such as the `Audit Log Create`.
+You do not need to configure the tenant settings for any system level webhooks, such as the `Audit Log Create`.
```

Comment:
```suggestion
You do not need to configure the tenant settings for any system level webhooks, such as the `Audit Log Create` webhook.
```

Reply:
Noted your suggestion and called it 'event' for clarity.